src/lib/components/InputGroup/InputGroup.jsx

```diff
@@ -0,0 +1,133 @@
+import PropTypes from 'prop-types';
+import React, { useContext } from 'react';
+import { Text } from '../Text';
+import { withGlobalProps } from '../../provider';
+import { classNames } from '../../utils/classNames';
+import { getRootValidationStateClassName } from '../_helpers/getRootValidationStateClassName';
+import { isChildrenEmpty } from '../_helpers/isChildrenEmpty';
+import { resolveContextOrProp } from '../_helpers/resolveContextOrProp';
+import { transferProps } from '../_helpers/transferProps';
+import { FormLayoutContext } from '../FormLayout';
+import { InputGroupContext } from './InputGroupContext';
+import styles from './InputGroup.scss';
+
+export const InputGroup = ({
```

Comment:
<img width="507" alt="image" src="https://user-images.githubusercontent.com/10213870/212653753-9e0272dc-8d7d-47f1-97a0-007489d8ae75.png">

Please fix alignment of labels to be horizontally centered.